src/content/monitoring/_index.en.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Monitoring"
+date = 2020-08-12T16:02:00+02:00
+weight = 20
+pre = "<b>4. </b>"
++++
+
+## Basic Overview
+
+Submariner provides a number of Prometheus metrics, and sets up `ServiceMonitor` instances which allow these metrics to be scraped by an
+in-cluster Prometheus instance.
```

Comment:
```suggestion
Submariner provides a number of metrics made available by `ServiceMonitor` instances that can consumed by an in-cluster Prometheus instance.
```

Reply:
The distinction is relevant IMO: Submariner provides metrics, and it also provides the configuration for an in-cluster Prometheus. The metrics can be used without the `ServiceMonitor`, and will be in many cases. “Scraping” is the appropriate term in Prometheus jargon.